### PR TITLE
doc: update and move READMEs

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**
to reflect new bazel prerequisite and the fact that two tools, autosynth and synthtool, now live in this repo.

Fixes https://github.com/googleapis/synthtool/issues/511

### CODE REVIEW

**Comment by SurferJeffAtGoogle** [comment]
> can you merge with master?

Done.

File: synthtool/README.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+| AB |
+
+| PR2 |
+| - |
+| XY |
+
+
+## Helpful tips
+### Where does the generated code go?
+SynthTool will run [Artman](https://hub.docker.com/r/googleapis/artman) which will create generated code that
```
Comment (elharo) [inline synthtool/README.md:384]: will run --> runs
will create --> creates
per https://developers.google.com/style/tense

Comment (SurferJeffAtGoogle) [reply synthtool/README.md]: Done.

File: synthtool/README.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+```
+
+## Basic usage
+To start the process of generation, clone the destination repository.
+```
+git clone git@github.com:googleapis/google-cloud-python.git
+cd google-cloud-python/
+```
+
+Navigate to the destination directory to generate the library.
```
Comment (elharo) [inline synthtool/README.md:33]: delete " to generate the library"

File: synthtool/README.md [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+
+Find examples below in different programming languages (Cloud Tasks API used as an example).
+
+### Python
+- Clone the destination repository:
+  ```
+  git clone git@github.com:googleapis/google-cloud-python.git
+  cd google-cloud-python/
+  ```
+- Navigate to the destination directory to generate the library:
```
Comment (elharo) [inline synthtool/README.md:65]: delete " to generate the library"

File: synthtool/README.md [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+[node_tasks_synth_py]: https://github.com/googleapis/nodejs-tasks/blob/master/synth.py
+[node_tasks_contributing]: https://github.com/googleapis/nodejs-tasks/blob/master/CONTRIBUTING.md
+
+### PHP
+- Clone the destination repository:
+  ```
+  git clone git@github.com:googleapis/google-cloud-php.git
+  cd google-cloud-php/
+  ```
+- Navigate to the destination directory to generate the library:
```
Comment (elharo) [inline synthtool/README.md:123]: delete " to generate the library"

File: synthtool/README.md [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+- See the Ruby [Contributing Guide][ruby_contributing]
+  or instructions to install dependencies, run tests, and submit a contribution.
+
+[ruby_tasks_library]: https://github.com/googleapis/google-cloud-ruby/tree/master/google-cloud-tasks
+[ruby_tasks_synth_py]: https://github.com/googleapis/google-cloud-ruby/blob/master/google-cloud-tasks/synth.py
+[ruby_contributing]: https://github.com/googleapis/google-cloud-ruby/blob/master/.github/CONTRIBUTING.md
+
+## Features
+### Templating
+SynthTool supports template files using [Jinja](http://jinja.pocoo.org/).
```
Comment (elharo) [inline synthtool/README.md:163]: https?

File: synthtool/README.md [hunk 6]
```diff
@@ -0,0 +1,386 @@
...
+    Previous versions:
+      {% for ver in previous_versions %}
+      - {{ ver }}
+      {% endfor %}
+    {% endif %}
+    {% endif %}
+    ```
+
+You can learn more about Jinga templating in the
+[Template Designer Documentation](http://jinja.pocoo.org/docs/templates/).
```
Comment (elharo) [inline synthtool/README.md:209]: https

File: synthtool/README.md [hunk 7]
```diff
@@ -0,0 +1,386 @@
...
+
+Don't forget to compile `gapic-generator` before running SynthTool.
+
+```
+cd path/to/local/gapic-generator
+./gradlew fatJar
+```
+
+### Local Template Files
+SynthTool supports specifying a local directory of templates. Specify the patht to the root
```
Comment (elharo) [inline synthtool/README.md:288]: patht --> path

File: synthtool/README.md [hunk 8]
```diff
@@ -0,0 +1,386 @@
...
+3. **Docker**  Some synth.py files require [Docker] to generate code.
+
+[python_downloads]: https://www.python.org/downloads/
+[pyenv]: https://github.com/pyenv/pyenv
+[Docker]: https://docs.docker.com/v17.09/engine/installation/#desktop
+
+## Add synthtool to your PYTHONPATH
+
+```
+..\synthtool$ export PYTHONPATH=`realpath .`
```
Comment (elharo) [inline synthtool/README.md:23]: What's the ..\ here?

Comment (SurferJeffAtGoogle) [reply synthtool/README.md:23]: It's intended to be read as an ellipsis:
some-path-on-your-computer\synthtool$

Comment (elharo) [reply synthtool/README.md:23]: Not obvious, I'm afraid.  I think just `$` is what most tech docs use in this context. Not everyone has their bash prompt set to show the directory.

Comment (SurferJeffAtGoogle) [reply synthtool/README.md]: I compared to the other commands in this doc, and they don't even have a $.  So I removed ..\synthtool$ to make it consistent.

File: synthtool/README.md (hunk 2 above)
Comment (SurferJeffAtGoogle) [reply synthtool/README.md]: Done.

File: synthtool/README.md (hunk 3 above)
Comment (SurferJeffAtGoogle) [reply synthtool/README.md]: Done.

File: synthtool/README.md (hunk 7 above)
Comment (SurferJeffAtGoogle) [reply synthtool/README.md]: Done.

**Pull request merged**